index.js

```diff
@@ -187,11 +187,20 @@ function writeShim_ (src, to, opts) {
     '$basedir=Split-Path $MyInvocation.MyCommand.Definition -Parent\n' +
     '\n' +
     '$exe=""\n' +
+    (opts.nodePath ? '$env_node_path=$env:NODE_PATH\n' +
+      `$env:NODE_PATH="${opts.nodePath.replace(/:/g, ';').replace(/\//g, '\\')}"\n` : '') +
```

Comment:
so on Windows it will convert `C:\foo` to `C;\foo`?

Reply:
I’ll get on fixing that, as I forgot about that case.